validateOffload.py

```diff
@@ -11,7 +11,8 @@
 from logger import logger
 import time
 from testConfig import TestConfig
-from iperf import IperfServer, IperfClient, EXTERNAL_IPERF3_SERVER
+from iperf import IperfServer, IperfClient, EXTERNAL_PERF_SERVER
```

Comment:
EXTERNAL_PERF_SERVER is imported from perf now

Reply:
Thanks for catching this.